src/components/PeriodicReports/reports.ts

```diff
@@ -0,0 +1,14 @@
+import { PeriodicReportFragment } from './PeriodicReport.graphql';
+
+type Report = Pick<PeriodicReportFragment, 'start' | 'end'>;
+
+export const getReportLabel = (report?: Report) => {
```

Comment:
My initial recommendation was to be in the `~/common/report.ts` file to be able to use it in multiple places where needed. But I strongly disagree that it should be in the `ReportLabel.tsx` file since we are using it for the title as well.

Reply:
I don't have a strong opinion on this one.  I am glad to put it where you feel it makes most sense.
I misunderstood your previous comment about this one which is why I put it in the periodicReports folder.